### PR TITLE
change: Fixed duplicate test in change_test.py

### PR DESCRIPTION
**Pull request by RJTK**
There were two identical tests.  I modified this one to be more reflective of the name.

### CODE REVIEW

**Comment by m-a-ge** [comment]
@RJTK, thanks a lot for spotting this!

File: exercises/change/change_test.py [hunk 1]
```diff
@@ -19,8 +19,8 @@ def test_change_with_Lower_Elbonia_Coins(self):
                          [21, 21, 21])
 
     def test_large_target_values(self):
-        self.assertEqual(find_minimum_coins(21, [2, 5, 10, 20, 50]),
-                         [2, 2, 2, 5, 10])
+        self.assertEqual(find_minimum_coins(219, [1, 5, 10, 20, 50]),
```
Comment (m-a-ge) [inline exercises/change/change_test.py]: Will it be possible to update values to match `canonical-data.json` for the exercise? - https://github.com/exercism/problem-specifications/blob/master/exercises/change/canonical-data.json

```
    {
      "description": "large target values",
      "property": "findFewestCoins",
      "coins": [1, 2, 5, 10, 20, 50, 100],
      "target": 999,
      "expected": [   2,   2,   5,  20,  20
                  ,  50, 100, 100, 100, 100
                  , 100, 100, 100, 100, 100 ]
    },
```

**Comment by RJTK** [comment]
I updated the Python file to match the json file, since it had no duplicates.

**Comment by m-a-ge** [comment]
@RJTK, thanks a lot for spotting and fixing this!

**Pull request merged**